TRANSLATING.md

```diff
@@ -91,10 +98,10 @@ If you spot something on [docs.astro.build](https://docs.astro.build/) that you
 
     ➤ Go to `src/pages/{language}/{page-slug}.md`
 
-## Contributing to translations
+# Contributing to translations
 
 Please see [CONTRIBUTING.md](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) for information about contributing via a fork, our Style Guide, and more!
----
```

Comment:
I wonder if we want to move all the Translation Tips _above_ “Adding a new language”? That section is really only aimed at maintainers, so might be better to relegate it to the bottom of the document.

Reply:
Yeah, "Adding a new language" is more of a one-time thing, but Review/Translation Tips can be revisited many times by contributors, so I agree with moving these up!